proto/options.proto

```diff
@@ -68,6 +68,25 @@ message AvoidEdge {
   }
 }
 
+/**
+ * Hierarchy limits controls expansion and transitions between hierarchy
+ * levels. It allows limiting expansion on hierarchy levels once
+ * some number of "upward" transitions have been made (e.g. from the local
+ * level to the arterial level). Expansion on a level is allowed within some
+ * distance from the destination location. This also allows control of where
+ * upward and downward transitions are allowed based on distance from the
+ * destination.
+ */
+message HierarchyLimits {
+ uint32 up_transition_count = 1;   // Keep track of # of upward transitions
```

Comment:
Using this for internal state is not ideal, I get that, but I don't want to have a wrapper class just for this attribute.